src/OpenTelemetry.AutoInstrumentation/Instrumentation.cs

```diff
@@ -187,7 +187,7 @@ public static void Initialize()
                 def.Dispose();
             }
 
-            Logger.Information<int>("The profiler has been initialized with {count} definitions.", payload.Definitions.Length);
+            Logger.Information<int>("The profiler has been initialized with {0} definitions.", payload.Definitions.Length);
```

Comment:
Side note. I am curious why it is a generic method. Why it cannot be simply something like:

```cs
Logger.Information("The profiler has been initialized with {0} definitions.", payload.Definitions.Length);
```

or

```cs
Logger.InformationFormat("The profiler has been initialized with {0} definitions.", payload.Definitions.Length);
```

Reply:
Likely it is an "optimization": instead of having only the signature with `params` object you have some signatures with a fixed number of parameters and generic types some restricted to classes and other signatures to structs. Together this avoids boxing value types like int and allocating an array for the params call.